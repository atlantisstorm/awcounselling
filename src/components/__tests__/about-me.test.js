import React from 'react';
import { render, fireEvent } from "@testing-library/react";
import AboutMe from '../about-me';

describe('AboutMe component', () => {
  it('should match snapshot', () => {
    const onClick = jest.fn();
    const { asFragment } = render(
      <AboutMe onClick={ onClick } />
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it('should contain contact opton link', () => {
    const onClick = jest.fn();
    const { getByTestId } = render(
      <AboutMe onClick={ onClick } />
    );

    const link = getByTestId("about-me--family-programme");
    fireEvent.click(link);
    expect(onClick).toHaveBeenCalledTimes(1);

    const dataName = link.getAttribute("data-name");
    expect(dataName).toBe("family-programme");
  });
});