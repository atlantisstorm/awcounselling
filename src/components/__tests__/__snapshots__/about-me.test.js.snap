// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe component should match snapshot 1`] = `
<DocumentFragment>
  <h5>
    About me
  </h5>
  <p>
    I am an accredited counsellor with the A.C.I.  (Addiction Counsellors of Ireland) and an affiliate member of the I.A.C.P.  (Irish Association for Counselling and Psychotherapy)  and work to their code of ethics.
  </p>
  <p>
    From Tuesday the1st of September 2020 I am  returning to  Co. Fermanagh  to work in my home based private practice which is located 9 miles from Enniskillen in Maguiresbridge,   Co. Fermanagh - see 
    <a
      data-name="contact"
      href="/"
    >
      contact
    </a>
     page for location.
  </p>
  <p>
    It is my intention to bring my Education / Family programme to as many people as possible either in group settings or remotely  via zoom See 
    <a
      data-name="family-programme"
      href="/"
    >
      Family programme
    </a>
     page full details.
  </p>
  <p>
    From 2017 to August 2020 I have worked in Smarmore Castle Private Clinic outside Ardee, Co.  Louth supporting patients through their treatment programme and while there was promoted to Specialist Therapist for Family Support.
  </p>
  <p>
    From 2013 to 2017 I worked in my private practice outside the village of Rosslea Co. Fermanagh along with working as a sessional counsellor for the following organizations 
  </p>
  <p>
    Nexus NI in Enniskillen which supports Adult Survivors of Sexual Abuse
  </p>
  <p>
    Action Mental Health New Horizons Enniskillen where I developed and facilitated the Impact of Alcohol Programme called the Time Out Programme which is Accredited with 10CPD points from the A.C.I. along with doing one to one support with service users.  I also facilitated a Womenâ€™s group there which proved life changing for many of the group members.
  </p>
  <p>
    Aisling Centre Enniskillen where I worked as an outreach counsellor for the Probation Service of NI.  Doing one to one counselling in the centre and facilitating the Journeying Together Family Support Group every two weeks for family members of all addictions.
  </p>
  <p>
    I facilitated my Education / Family Programme in several locations in Co. Fermanagh and surrounding counties
  </p>
  <p>
    In 2013 I returned home after working three years in Derry City in the Y.A.R.D. Project as the project counsellor and in White Oaks Rehabilitation Centre in Donegal supporting residents through their 30 day programme.
  </p>
  <p>
    I also developed and facilitated a one day 
    <a
      data-name="family-programme"
      data-testid="about-me--family-programme"
      href="/"
    >
      Education/Family programme
    </a>
     for the family, friends, people who live, work or have lived with people who use or abuse alcohol, drugs or and gambling that I ran on a regular basis in Columba House, Queen St, Derry and this year out of White Oaks Rehabilitation Centre. This programme is accredited with 6 CPD points from the A.C.I. 
  </p>
  <p>
    I also completed a one year clinical placement with Nexus Institute Derry where I counselled adult survivors of sexual abuse.
  </p>
  <p>
    During my training I spent nine months working in Aiseiri Treatment Centre in Wexford.
  </p>
  <p>
    I co-facilitated pre-marriage courses for three years with Accord in Enniskillen. 
  </p>
  <p>
    I am also an Al-Anon Family Group member for over twenty Years.
  </p>
  <p>
    I have professional liability insurance.
  </p>
</DocumentFragment>
`;
